modules/installation-configuration-parameters.adoc

```diff
@@ -1284,6 +1283,52 @@ Optional VMware vSphere machine pool configuration parameters are described in t
 
 endif::vsphere,vmc[]
 
+ifdef::ash-default[]
+[id="installation-configuration-parameters-additional-azure-stack-hub_{context}"]
+== Additional Azure Stack Hub configuration parameters
+
+Additional Azure configuration parameters are described in the following table:
+
+.Additional Azure Stack Hub parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.azure.armEndpoint`
+|The URL of the Azure Resource Manager endpoint that your Azure Stack Hub operator provides.
+|String
+
+|`platform.azure.baseDomainResourceGroupName`
+|The name of the resource group that contains the DNS zone for your base domain.
+|String, for example `production_cluster`.
+
+|`platform.azure.region`
+|The name of your Azure Stack Hub local region.
+|String
+
+|`platform.azure.resourceGroupName`
+| The name of an already existing resource group to install your cluster to. If undefined, a new resource group is created for the cluster.
+|String, for example `existing_resource_group`.
+
+|`platform.azure.outboundType`
+|The outbound routing strategy used to connect your cluster to the internet. If
+you are using user-defined routing, you must have pre-existing networking
+available where the outbound routing has already been configured prior to
+installing a cluster. The installation program is not responsible for
+configuring user-defined routing.
+|`LoadBalancer` or `UserDefinedRouting`. The default is `LoadBalancer`.
+
+|`platform.azure.cloudName`
+|The name of the Azure cloud environment that is used to configure the Azure SDK with the appropriate Azure API endpoints.
+|`AzureStackCloud`
+
+|`clusterOSImage`
+|The url of a storage blob in the Azure Stack environment that contains an RHCOS VHD.
```

Comment:
```suggestion
|The URL of a storage blob in the Azure Stack environment that contains an RHCOS VHD.
```

Reply:
Done.

Reply:
:+1: 

I missed s/RHCOS/{op-system} (there are a few instances in this PR to change) - oops! :bow: